source/markdown/data_controle_2024.Rmd

```diff
@@ -0,0 +1,1572 @@
+
+# Controle finale dataset
+
+Nadat we de pipeline hebben doorlopen bekomen we de finale dataset.
+Ook hierop doen we enkele controlestappen.
+Zijn alle soorten correct (bv. Blauwe Pauw i.p.v. Blauwe Reiger)? Zijn er opvallend veel individuen opgegeven (bv. 300 i.p.v. 30)? ...
+
+```{r}
+mas_data_clean <- tar_read("mas_data_clean", store = targets_store)
+```
+
+## Telpunt- en waarneminglocaties
+
+Zie Sectie \@ref(controle-pipeline).
+
+## Timing observaties
+
+De data bevatten geen info over precieze tijdstippen van tellingen, enkel datums.
+De datums werden gecontroleerd in Subsectie \@ref(time-periods).
+
+## Aantal individuen
+
+In sommige gevallen worden meer dan 50 individuen gezien.
+
+```{r}
+mas_data_clean %>%
+  st_drop_geometry() %>%
+  filter(aantal >= 50) %>%
+  count(aantal, name = "aantal keer in dataset") %>%
+  rename("aantal vogels" = aantal) %>%
+  kable()
+```
+
+Een opvalllende is ook een uitschieter van 3000 individuen.
+We zien dat de meeste waarnemingen wel gaan over soorten die in groep voorkomen, zoals Spreeuw, Grauwe Gans, Roek, Houtduif ...
+
+```{r}
+mas_data_clean %>%
+  st_drop_geometry() %>%
+  select(aantal, naam, plotnaam, waarnemer, datum) %>%
+  filter(aantal >= 50) %>%
+  arrange(aantal) %>%
+  kable()
+```
+
+We schrijven deze waarnemingen weg zodat ze toch gecontroleerd kunnen worden.
+
+```{r}
+mas_data_clean %>%
+  filter(aantal >= 50) %>%
+  arrange(aantal) %>%
+  write_csv(file.path(mbag_dir, "output", "data_controle",
+                      "waarnemingen_met_hoge_aantallen.csv"))
+```
+
+## Soortnamen
+
+De volgende soorten werden gezien.
+Deze lijst moet gecontroleerd worden op vreemde namen die mogelijks fout zijn.
+
+```{r}
+mas_data_clean %>%
+  st_drop_geometry() %>%
+  count(naam, name =  "aantal waarnemingen") %>%
+  arrange(naam) %>%
+  kable()
+```
+
+We schrijven de data voor Waterspreeuw weg.
+
+```{r}
+mas_data_clean %>%
+  st_drop_geometry() %>%
+  filter(naam == "Waterspreeuw") %>%
+  write_csv(file.path(mbag_dir, "output", "data_controle",
+                      "vreemde_soorten_mastellingen.csv"))
+```
+
+## Broedcodes
+
+Broedcodes zijn ook belangrijk voor verdere data analyse.
+
+Communicatie Simon Desmet:
+
+*In de Avimap-versie voor Android kan een teller enkel het aantal wijzigen als de vogel broedcode 0 krijgt, maar met een iPhone is dat wel mogelijk bij vogels die met een hogere broedcode dan 0 ingevoerd werden. Sommige tellers hebben dit blijkbaar effectief gedaan, hoewel wij op voorhand gevraagd hadden dit niet te doen.*
+
+Het is echter moeilijk om deze te controleren.
+Wel kunnen we de aantallen visualiseren.
+
+```{r}
+mas_data_clean %>%
+  st_drop_geometry() %>%
+  count(wrntype, wrntype_omschrijving, name = "n_obs") %>%
+  rename(
+    "broedcode" = wrntype,
+    "omschrijving" = wrntype_omschrijving,
+  ) %>%
+  kable()
+```
+
+**Broedcode 1-5**
+
+Er is inderdaad een aantal keer hogere aantallen dan 1 ingevoerd bij broedcodes 1-5.
+
```

Comment:
This check is very important. There are appear to be many problematic cases. How will they be corrected? In the sovon database? We need to be careful how to correct this. I think we discussed this before, but the database is a bit misleading in this regard. When broedcode == 0, the number of **individuals** is counted, whereas in case of broedcode > 0, the agreement is to set this number to 1, but it represents a bird couple: i.e. 2 individuals. This is why so many field workers have change 1 to 2. It is a pity that two different counting units are mixed in the same field. I'm not even sure if we have dealt with this appropriately in our analyses.

Reply:
terugkoppelen met Johannes, eens kijken naar jaren in controle bestand

Reply:
done in: [ee880d6](https://github.com/inbo/mbag-mas/pull/37/commits/ee880d69c4c7de32bb4e3b6ead058035ec6bada2)